src/alchemlyb/workflows/abfe.py

```diff
@@ -61,6 +62,8 @@ class ABFE(WorkflowBase):
 
 
     .. versionadded:: 1.0.0
+    .. versionchanged:: 2.0.1
+        The `dir` argument expect a real directory and wildcard are no longer accepted.
```

Comment:
We should not have done this API change in 2.0.1 as this breaks semver. 

However, the change is small enough so I am ok with just moving forward.

Reply:
Sorry, I hadn't thought of this before I do this change.